poetry.lock

```diff
@@ -2032,51 +2032,71 @@ files = [
 
 [[package]]
 name = "markupsafe"
-version = "2.1.0"
+version = "2.1.5"
```

Comment:
it seems like markupsafe is still in lock file even though it gets removed from toml?

Reply:
This is intended, now that it's not in the toml file, poetry will still add it to the lockfile since it's a dependancy. But the verison isn't locked so poetry is free to add the version that is needed based on other dependancies.